kolibri/plugins/learn/assets/src/composables/useSearch.js

```diff
@@ -350,7 +339,7 @@ export default function useSearch(descendant, store, router) {
   // component to setup the available labels for child components
   // to consume them.
 
-  const otherDeviceGlobalLabels = ref(null);
+  const globalLabels = ref(null);
```

Comment:
Correct - previously we were treating the local device search labels specially because we bootstrapped them into the page and loaded them from plugin_data, however, that's not compatible with them changing during the course of the page lifetime (as they will need to when resources are being dynamically downloaded by the learner).